src/vmem.c

```diff
@@ -282,6 +289,28 @@ static int vmem_read (void)
     submit_two (NULL, "vmpage_io", "swap", pswpin, pswpout);
 #endif /* KERNEL_LINUX */
```

Comment:
isn't this the one that should have a comment || KERNEL_DARWIN? and the above comment just removed?